elixir/apps/web/lib/web/live/clients/show.ex

```diff
@@ -112,32 +114,31 @@ defmodule Web.Clients.Show do
             </:value>
           </.vertical_table_row>
           <.vertical_table_row>
-            <:label>Last used sign in method</:label>
+            <:label>Current sign in method</:label>
             <:value>
-              <div :if={@client.actor.type != :service_account} class="flex items-center">
-                <.identity_identifier account={@account} identity={@client.last_used_token.identity} />
+              <div
+                :if={@current_token && @client.actor.type != :service_account}
+                class="flex items-center"
+              >
+                <.identity_identifier account={@account} identity={@current_token.identity} />
                 <.link
-                  navigate={
-                    ~p"/#{@account}/actors/#{@client.actor_id}?#tokens-#{@client.last_used_token_id}"
-                  }
+                  navigate={~p"/#{@account}/actors/#{@client.actor_id}?#tokens-#{@current_token.id}"}
                   class={[link_style(), "text-xs"]}
                 >
                   show tokens
                 </.link>
               </div>
-              <div :if={@client.actor.type == :service_account}>
+              <div :if={@current_token && @client.actor.type == :service_account}>
                 token
                 <.link
-                  navigate={
-                    ~p"/#{@account}/actors/#{@client.actor_id}?#tokens-#{@client.last_used_token_id}"
-                  }
+                  navigate={~p"/#{@account}/actors/#{@client.actor_id}?#tokens-#{@current_token.id}"}
                   class={[link_style()]}
                 >
-                  {@client.last_used_token.name}
+                  {@current_token.name}
                 </.link>
-                <span :if={not is_nil(@client.last_used_token.deleted_at)}>
-                  (deleted)
-                </span>
+              </div>
+              <div :if={!@current_token} class="text-gray-500">
+                Client is offline - sign in method unavailable
```

Comment:
Would it make sense just to hide this row if the client is not signed in.

Reply:
If this were just an icon on the page somewhere I would say yes, however, since this is a row in a table, I feel like it would be better to have the table be consistent.  Thoughts?